{
  "uid" : "eb6be8439f05e9fe",
  "name" : "TC 01.03.08 correctness of the \"Hoodies & Sweatshirts\" link of the \"Women\" section",
  "fullName" : "tests.test_header.TestHeader#test_tc_01_03_08_correctness_of_hoodies_women_section_link",
  "historyId" : "f4a77ee2190d52fad521d519a2b46b4e",
  "time" : {
    "start" : 1697334126369,
    "stop" : 1697334131710,
    "duration" : 5341
  },
  "description" : "Check 'Hoodies & Sweatshirts' subsection link click in 'Women' section redirects to the         Hoodies & Sweatshirts-women page and the 'Hoodies & Sweatshirts' header is displayed",
  "descriptionHtml" : "<p>Check 'Hoodies &amp; Sweatshirts' subsection link click in 'Women' section redirects to the         Hoodies &amp; Sweatshirts-women page and the 'Hoodies &amp; Sweatshirts' header is displayed</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5631c65216b3 <unknown>\n#1 0x5631c61f71e7 <unknown>\n#2 0x5631c623e566 <unknown>\n#3 0x5631c623e651 <unknown>\n#4 0x5631c627ba74 <unknown>\n#5 0x5631c62600ed <unknown>\n#6 0x5631c6279322 <unknown>\n#7 0x5631c625fe93 <unknown>\n#8 0x5631c6232934 <unknown>\n#9 0x5631c623371e <unknown>\n#10 0x5631c64e6cb8 <unknown>\n#11 0x5631c64eabf0 <unknown>\n#12 0x5631c64f519c <unknown>\n#13 0x5631c64eb808 <unknown>\n#14 0x5631c64b827f <unknown>\n#15 0x5631c650fe88 <unknown>\n#16 0x5631c6510059 <unknown>\n#17 0x5631c6520843 <unknown>\n#18 0x7f50bc094b43 <unknown>",
  "statusTrace" : "self = <test_header.TestHeader object at 0x7fe0af35c190>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9ec5bc394eb880f0f7b8ffa6b640d9a9\")>\n\n    @allure.title('TC 01.03.08 correctness of the \"Hoodies & Sweatshirts\" link of the \"Women\" section')\n    def test_tc_01_03_08_correctness_of_hoodies_women_section_link(self, driver):\n        \"\"\"Check 'Hoodies & Sweatshirts' subsection link click in 'Women' section redirects to the \\\n        Hoodies & Sweatshirts-women page and the 'Hoodies & Sweatshirts' header is displayed\"\"\"\n        page = HeaderPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       page.check_redirection_of_hoodies_subsection_of_women_section_link()\n\ntests/test_header.py:463: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/header_page.py:362: in check_redirection_of_hoodies_subsection_of_women_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.WOMEN_SECTION))\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9ec5bc394eb880f0f7b8ffa6b640d9a9\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fe0b0a7e520>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5631c65216b3 <unknown>\nE       #1 0x5631c61f71e7 <unknown>\nE       #2 0x5631c623e566 <unknown>\nE       #3 0x5631c623e651 <unknown>\nE       #4 0x5631c627ba74 <unknown>\nE       #5 0x5631c62600ed <unknown>\nE       #6 0x5631c6279322 <unknown>\nE       #7 0x5631c625fe93 <unknown>\nE       #8 0x5631c6232934 <unknown>\nE       #9 0x5631c623371e <unknown>\nE       #10 0x5631c64e6cb8 <unknown>\nE       #11 0x5631c64eabf0 <unknown>\nE       #12 0x5631c64f519c <unknown>\nE       #13 0x5631c64eb808 <unknown>\nE       #14 0x5631c64b827f <unknown>\nE       #15 0x5631c650fe88 <unknown>\nE       #16 0x5631c6510059 <unknown>\nE       #17 0x5631c6520843 <unknown>\nE       #18 0x7f50bc094b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1697334125235,
      "stop" : 1697334126369,
      "duration" : 1134
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1697332188525,
      "stop" : 1697332188634,
      "duration" : 109
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1697332188634,
      "stop" : 1697332188634,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Check 'Hoodies & Sweatshirts' subsection link click in 'Women' section redirects to the         Hoodies & Sweatshirts-women page and the 'Hoodies & Sweatshirts' header is displayed",
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5631c65216b3 <unknown>\n#1 0x5631c61f71e7 <unknown>\n#2 0x5631c623e566 <unknown>\n#3 0x5631c623e651 <unknown>\n#4 0x5631c627ba74 <unknown>\n#5 0x5631c62600ed <unknown>\n#6 0x5631c6279322 <unknown>\n#7 0x5631c625fe93 <unknown>\n#8 0x5631c6232934 <unknown>\n#9 0x5631c623371e <unknown>\n#10 0x5631c64e6cb8 <unknown>\n#11 0x5631c64eabf0 <unknown>\n#12 0x5631c64f519c <unknown>\n#13 0x5631c64eb808 <unknown>\n#14 0x5631c64b827f <unknown>\n#15 0x5631c650fe88 <unknown>\n#16 0x5631c6510059 <unknown>\n#17 0x5631c6520843 <unknown>\n#18 0x7f50bc094b43 <unknown>",
    "statusTrace" : "self = <test_header.TestHeader object at 0x7fe0af35c190>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9ec5bc394eb880f0f7b8ffa6b640d9a9\")>\n\n    @allure.title('TC 01.03.08 correctness of the \"Hoodies & Sweatshirts\" link of the \"Women\" section')\n    def test_tc_01_03_08_correctness_of_hoodies_women_section_link(self, driver):\n        \"\"\"Check 'Hoodies & Sweatshirts' subsection link click in 'Women' section redirects to the \\\n        Hoodies & Sweatshirts-women page and the 'Hoodies & Sweatshirts' header is displayed\"\"\"\n        page = HeaderPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       page.check_redirection_of_hoodies_subsection_of_women_section_link()\n\ntests/test_header.py:463: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/header_page.py:362: in check_redirection_of_hoodies_subsection_of_women_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.WOMEN_SECTION))\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9ec5bc394eb880f0f7b8ffa6b640d9a9\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fe0b0a7e520>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5631c65216b3 <unknown>\nE       #1 0x5631c61f71e7 <unknown>\nE       #2 0x5631c623e566 <unknown>\nE       #3 0x5631c623e651 <unknown>\nE       #4 0x5631c627ba74 <unknown>\nE       #5 0x5631c62600ed <unknown>\nE       #6 0x5631c6279322 <unknown>\nE       #7 0x5631c625fe93 <unknown>\nE       #8 0x5631c6232934 <unknown>\nE       #9 0x5631c623371e <unknown>\nE       #10 0x5631c64e6cb8 <unknown>\nE       #11 0x5631c64eabf0 <unknown>\nE       #12 0x5631c64f519c <unknown>\nE       #13 0x5631c64eb808 <unknown>\nE       #14 0x5631c64b827f <unknown>\nE       #15 0x5631c650fe88 <unknown>\nE       #16 0x5631c6510059 <unknown>\nE       #17 0x5631c6520843 <unknown>\nE       #18 0x7f50bc094b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1697334126369,
        "stop" : 1697334126581,
        "duration" : 212
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Check redirection of the \"Hoodies & Sweatshirts\" subsection of the \"Women\" section link",
      "time" : {
        "start" : 1697334126581,
        "stop" : 1697334131709,
        "duration" : 5128
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5631c65216b3 <unknown>\n#1 0x5631c61f71e7 <unknown>\n#2 0x5631c623e566 <unknown>\n#3 0x5631c623e651 <unknown>\n#4 0x5631c627ba74 <unknown>\n#5 0x5631c62600ed <unknown>\n#6 0x5631c6279322 <unknown>\n#7 0x5631c625fe93 <unknown>\n#8 0x5631c6232934 <unknown>\n#9 0x5631c623371e <unknown>\n#10 0x5631c64e6cb8 <unknown>\n#11 0x5631c64eabf0 <unknown>\n#12 0x5631c64f519c <unknown>\n#13 0x5631c64eb808 <unknown>\n#14 0x5631c64b827f <unknown>\n#15 0x5631c650fe88 <unknown>\n#16 0x5631c6510059 <unknown>\n#17 0x5631c6520843 <unknown>\n#18 0x7f50bc094b43 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/header_page.py\", line 362, in check_redirection_of_hoodies_subsection_of_women_section_link\n    self.action_move_to_element(self.element_is_visible(self.header_locators.WOMEN_SECTION))\n                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Find a visible element",
        "time" : {
          "start" : 1697334126581,
          "stop" : 1697334131709,
          "duration" : 5128
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5631c65216b3 <unknown>\n#1 0x5631c61f71e7 <unknown>\n#2 0x5631c623e566 <unknown>\n#3 0x5631c623e651 <unknown>\n#4 0x5631c627ba74 <unknown>\n#5 0x5631c62600ed <unknown>\n#6 0x5631c6279322 <unknown>\n#7 0x5631c625fe93 <unknown>\n#8 0x5631c6232934 <unknown>\n#9 0x5631c623371e <unknown>\n#10 0x5631c64e6cb8 <unknown>\n#11 0x5631c64eabf0 <unknown>\n#12 0x5631c64f519c <unknown>\n#13 0x5631c64eb808 <unknown>\n#14 0x5631c64b827f <unknown>\n#15 0x5631c650fe88 <unknown>\n#16 0x5631c6510059 <unknown>\n#17 0x5631c6520843 <unknown>\n#18 0x7f50bc094b43 <unknown>\n\n",
        "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ {
          "name" : "Find a present element",
          "time" : {
            "start" : 1697334126581,
            "stop" : 1697334131709,
            "duration" : 5128
          },
          "status" : "broken",
          "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5631c65216b3 <unknown>\n#1 0x5631c61f71e7 <unknown>\n#2 0x5631c623e566 <unknown>\n#3 0x5631c623e651 <unknown>\n#4 0x5631c627ba74 <unknown>\n#5 0x5631c62600ed <unknown>\n#6 0x5631c6279322 <unknown>\n#7 0x5631c625fe93 <unknown>\n#8 0x5631c6232934 <unknown>\n#9 0x5631c623371e <unknown>\n#10 0x5631c64e6cb8 <unknown>\n#11 0x5631c64eabf0 <unknown>\n#12 0x5631c64f519c <unknown>\n#13 0x5631c64eb808 <unknown>\n#14 0x5631c64b827f <unknown>\n#15 0x5631c650fe88 <unknown>\n#16 0x5631c6510059 <unknown>\n#17 0x5631c6520843 <unknown>\n#18 0x7f50bc094b43 <unknown>\n\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "locator",
            "value" : "('xpath', \"(//a[@id='ui-id-4'])[1]\")"
          }, {
            "name" : "timeout",
            "value" : "5"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "locator",
          "value" : "('xpath', \"(//a[@id='ui-id-4'])[1]\")"
        }, {
          "name" : "timeout",
          "value" : "10"
        } ],
        "stepsCount" : 1,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 2,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "fec0cddc9805c584",
      "name" : "Screenshot",
      "source" : "fec0cddc9805c584.png",
      "type" : "image/png",
      "size" : 14900
    }, {
      "uid" : "45caf94905803e2",
      "name" : "HTML source",
      "source" : "45caf94905803e2.html",
      "type" : "text/html",
      "size" : 571
    } ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1697334131879,
      "stop" : 1697334131939,
      "duration" : 60
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1697335158583,
      "stop" : 1697335158584,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_header"
  }, {
    "name" : "subSuite",
    "value" : "TestHeader"
  }, {
    "name" : "host",
    "value" : "fv-az163-423"
  }, {
    "name" : "thread",
    "value" : "3041-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_header"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "eb6be8439f05e9fe.json",
  "parameterValues" : [ ]
}